eppy/EPlusInterfaceFunctions/parse_idd.py

```diff
@@ -78,13 +79,9 @@ def removeblanklines(astr):
     lines = astr.splitlines()
     lines = [line for line in lines if line.strip() != ""]
     return "\n".join(lines)
-
-@decorator
-def embeddgroupdata(extract_func, fname, debug=False):
-    """insert group info into extracted idd"""
     
-    # copied the reading of fname from extractidddata - start
-    # removed decode (the decode will happen in extractidddata)
+def _readfname(fname):
```

Comment:
It is is a snippet copied from the function `extractidddata` which is further below in the code
I have changed the documentation to clarify that